### PR TITLE
[build/Android] Update android lib build script for PyTorch

### PR DESCRIPTION
**Pull request by anyj0527**
- Add PyTorch to build script for android library

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: api/android/build-android-lib.sh [hunk 1]
```diff
@@ -404,6 +425,13 @@ if [[ $enable_snpe == "yes" ]]; then
     cp $SNPE_DIRECTORY/lib/dsp/libsnpe*.so api/src/main/jni/snpe/lib/ext/arm64-v8a
 fi
 
+# Update PyTorch option
+if [[ $enable_pytorch == "yes" ]]; then
+    sed -i "s|ENABLE_PYTORCH := false|ENABLE_PYTORCH := true|" api/src/main/jni/Android.mk
+    sed -i "s|PYTORCH_VERSION := 1.8.0|PYTORCH_VERSION := $pytorch_ver|" api/src/main/jni/Android-tensorflow-lite.mk
```
Comment (jaeyun-jung) [inline api/android/build-android-lib.sh:431]: Fix this line to Android-pytorch.mk, not tf-lite mk file.

Comment (anyj0527) [reply api/android/build-android-lib.sh]: It's fixed. Thanks!

**Review by jaeyun-jung — APPROVED** [review]

File: api/android/build-android-lib.sh [hunk 2]
```diff
@@ -87,6 +90,12 @@ enable_nnfw="yes"
 # Enable SNPE
 enable_snpe="no"
 
+# Enable PyTorch
+enable_pytorch="no"
+
+#Set PyTorch version (available: 1.8.0 (unstable))
```
Comment (dongju-chae) [inline api/android/build-android-lib.sh:96]: Minor comment: need a space between '#' and 'S'.

File: api/android/build-android-lib.sh [hunk 3]
```diff
@@ -44,6 +44,9 @@
 ##@@   --enable_snpe=(yes|no)
 ##@@       'yes'      : build with sub-plugin for SNPE
 ##@@       'no'       : [default]
+##@@   --enable_pytorch=(yes|no)
```
Comment (dongju-chae) [inline api/android/build-android-lib.sh:47]: How about using the same way with `tflite_ver`?
For example, 
```(yes(:(1.8.0))?|no)``` and `default is 1.8.0`

File: api/android/build-android-lib.sh (hunk 2 above)
Comment (anyj0527) [reply api/android/build-android-lib.sh]: Thanks!

File: api/android/build-android-lib.sh (hunk 3 above)
Comment (anyj0527) [reply api/android/build-android-lib.sh]: I updated the code to after the `tflie_ver`. Thanks for suggesting!

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**